### ISSUE TITLE
Windows Install Problem

### ISSUE THREAD

**Issue by pgregory9** [cmhughes/latexindent.pl#176]
Hi,

I've searched all over for a solution, read the pdf, but cannot figure what's wrong. I'm usinf Windows 10. I installed latexindent via MikTeX, made sure it's in my %path%, but I cannot get latexindent to even show me the version number. Below is what I get.

I think it might be related to this issue, I am on 64bit machine: https://github.com/cmhughes/latexindent.pl/issues/60

Any advice will be greatly appreciated.

P

C:\Users\gregoryp\Box\My Documents\_Philosophy\Work\Logic\Final\Exercises>latexindent -v

Can't modify constant item in scalar assignment at C:/Users/gregoryp/AppData/Local/Programs/MiKTeX 2.9/scripts/latexindent/LatexIndent/Sentence.pm line 156, near "qr/$trailingCommentRegExp/s;"
Global symbol "$notWithinSentence" requires explicit package name at C:/Users/gregoryp/AppData/Local/Programs/MiKTeX 2.9/scripts/latexindent/LatexIndent/Sentence.pm line 168.
Global symbol "$notWithinSentence" requires explicit package name at C:/Users/gregoryp/AppData/Local/Programs/MiKTeX 2.9/scripts/latexindent/LatexIndent/Sentence.pm line 173.
Global symbol "$notWithinSentence" requires explicit package name at C:/Users/gregoryp/AppData/Local/Programs/MiKTeX 2.9/scripts/latexindent/LatexIndent/Sentence.pm line 180.
Compilation failed in require at C:/Users/gregoryp/AppData/Local/Programs/MiKTeX 2.9/scripts/latexindent/LatexIndent/Document.pm line 32.
BEGIN failed--compilation aborted at C:/Users/gregoryp/AppData/Local/Programs/MiKTeX 2.9/scripts/latexindent/LatexIndent/Document.pm line 32.
Compilation failed in require at C:\Users\gregoryp\AppData\Local\Programs\MiKTeX 2.9\scripts/latexindent\latexindent.pl line 27.
BEGIN failed--compilation aborted at C:\Users\gregoryp\AppData\Local\Programs\MiKTeX 2.9\scripts/latexindent\latexindent.pl line 27.

**Comment by cmhughes** [comment]
Hello! 
Thanks for this. What happens when you try to compile the minimal script in 

https://latexindentpl.readthedocs.io/en/latest/appendices.html#lst-helloworld

Let me know!

**Comment by pgregory9** [comment]
Hi!

I feel like I've wasted your time. I've found 2 solutions.

1. I installed Strawberry Perl, (re)created latexindent.exe via the instructions in the manual, made sure my %path% was set right, and it now works!

2. Since I now have perl working, I installed the modules via your instructions in the manual, make sure I use the full path to latexindent.pl with "perl" and it works that way too!

And yes. I was able to get helloworld.pl to work.

A good day of learning. Sorry I jumped the gun on opening an issue

P
